prog-o-meter.py

```diff
@@ -51,48 +52,54 @@ def __init__(self, _days, _filename, _username):
         self.button_layout()
         self.prog_o_meter()
         self.progress()
         self.root.mainloop()
     def canvas_layout(self):
         """Display a Tkinter canvas.
-        
-        Creates a 1200x600 px canvas, with a greeting text, and a text stating how many days the user have left untill they reach the goal.  
-       
+
+        Creates a 1200x600 px canvas, with a greeting text, and a text stating how many days the user have left untill they reach the goal.
+
         Attributes:
             CANVAS_WIDTH: The width of the canvas is hardcoded to 1200 px.
             CANVAS_HEIGHT: The height of the canvas is hardcoded to 600 px.
             canvas: The Tkinter canvas widget.
-            countdown_text: A text element on the canvas, stating how many days the user have left before reaching their goal. 
+            countdown_text: A text element on the canvas, stating how many days the user have left before reaching their goal.
         """
         self.CANVAS_WIDTH = 1200
         self.CANVAS_HEIGHT = 600
         VERTICAL_TEXT_POSITION = 100
         self.canvas = Tk.Canvas(self.root, width = self.CANVAS_WIDTH, height = self.CANVAS_HEIGHT)
         self.canvas.pack()
-        self.canvas.create_text(self.CANVAS_WIDTH/2, VERTICAL_TEXT_POSITION, text = ("".join(("Hello ", self.username))))
+        self.hog = self.canvas.create_text(self.CANVAS_WIDTH/2, VERTICAL_TEXT_POSITION, text = ("".join(("Hello ", self.username))))
         self.countdown_text = self.canvas.create_text(self.CANVAS_WIDTH/2, VERTICAL_TEXT_POSITION+40, justify = Tk.CENTER, text = "".join(("You have ", str(self.days_remaining), " days left!\n\n", "If you code everyday, you will be done with this project on ", self.completion_date)))
+        self.encourage_text = self.canvas.create_text(self.CANVAS_WIDTH/2, VERTICAL_TEXT_POSITION+40, justify = Tk.CENTER, text = "".join(self.encourage[self.new_no(self.x)]))
+
     def button_layout(self):
         """Display a button with the text "1 more day!" on the canvas.
-        
-        Creates and display a button with the text "1 more day!" with the function add_day() as callback function. If user have already reached their goal, the button is displayed, but is disabled. 
-       
+
+        Creates and display a button with the text "1 more day!" with the function add_day() as callback function. If user have already reached their goal, the button is displayed, but is disabled.
+
         Attributes:
             add_day_button: A button with the text "1 more day!", which calls the function add_day
         """
         self.add_day_button = Tk.Button(self.root, text = "1 more day!", command = self.add_day)
         self.add_day_button.pack()
         if self.days >= self.GOAL:        # Disable add_day_button if goal have been reached
             self.add_day_button.config(state = "disabled")
+            self.canvas.itemconfig(self.hog, text=("".join(("Congrats! ", self.username))))

```

Comment:
Cool! This is a nice addition, hadn't thought about changing this, when user reached 100 days :)